### ISSUE TITLE
[ARM] Missing instructions

### ISSUE THREAD

**Issue by cla7aye15I4nd** [unicorn-engine/unicorn#1514]
```python
from unicorn import *

# 13 fb 01 23  smlabb r3, r3, r1, r2
ins = b'\x13\xfb\x01\x23'

uc = Uc(UC_ARCH_ARM, UC_MODE_ARM + UC_MODE_MCLASS + UC_MODE_THUMB)

uc.mem_map(0x1000, 0x1000)
uc.mem_write(0x1000, ins)

uc.emu_start(0x1000 + 1, 0, count=1)
# unicorn.unicorn.UcError: Invalid instruction (UC_ERR_INSN_INVALID)
```

**Comment by wtdcode** [comment]
By default unicorn uses `cortex-m33` while `smlabb` belongs to `cortex-m7`. You may switch the CPU model by `uc_ctl_set_cpu_model`.

**Comment by cla7aye15I4nd** [comment]
I'm not sure if I handle right, but same error happened
```python
from unicorn import *
from unicorn.arm_const import *

# 13 fb 01 23  smlabb r3, r3, r1, r2
ins = b'\x13\xfb\x01\x23'

uc = Uc(UC_ARCH_ARM, UC_MODE_ARM + UC_MODE_MCLASS + UC_MODE_THUMB)
# uc.ctl_set_cpu_model(UC_CPU_ARM_CORTEX_M4)
uc.ctl_set_cpu_model(UC_CPU_ARM_CORTEX_M7)

uc.mem_map(0x1000, 0x1000)
uc.mem_write(0x1000, ins)

uc.emu_start(0x1000 + 1, 0, count=1)
# unicorn.unicorn.UcError: Invalid instruction (UC_ERR_INSN_INVALID)
```

**Comment by wtdcode** [comment]
Fixed in cddc9cf2ede8d6c388e85c3828aec8a3004cfdb0.

In addition, looks like `enable_vfp` in qiling is not needed any more after this fix.
